sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -479,6 +479,20 @@ void nav_home(void)
   nav_run();
 }
 
+/** Set manual roll, pitch and yaw without stabilization
+ *
+ * @param[in] roll The amount of roll between -1.0 and 1.0
```

Comment:
really? I don't think so...
the nav_roll/pitch/heading should be in radians in fixed point